HtmlDiff/Utils.cs

```diff
@@ -9,7 +9,7 @@ public static class Utils
         private static Regex closingTagTexRegex = new Regex("^\\s*</[^>]+>\\s*$", RegexOptions.Compiled);
         private static Regex tagWordRegex = new Regex(@"<[^\s>]+", RegexOptions.Compiled);
         private static Regex whitespaceRegex = new Regex("^(\\s|&nbsp;)+$", RegexOptions.Compiled);
-        private static Regex wordRegex = new Regex(@"[\w\#@]+", RegexOptions.Compiled | RegexOptions.ECMAScript);
+        private static Regex wordRegex = new Regex(@"[\w\#@]+", RegexOptions.ECMAScript);
```

Comment:
@caleblloyd To be safe, it might be worth look at using a conditional directive [as outlined here](https://docs.microsoft.com/en-us/dotnet/articles/core/tutorials/libraries) for the .NET Core version until the bug is resolved.